### PR TITLE
Density Fitting

### PR DESCRIPTION
**Pull request by carlosevmoura**
- Density-fitting support for CVS-IP-MR-ADC implemented
- Analysis tools for CVS-IP-MR-ADC implemented

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -15,4 +15,9 @@ Prism 0.2 (2023-05-12)
 Prism 0.2.1 (2023-07-04)
 ------------------------
 * Added
-  - Efficient combinations of einsum terms containing 3-RDMs and 4-RDMs
\ No newline at end of file
+  - Efficient combinations of einsum terms containing 3-RDMs and 4-RDMs
+
+Prism 0.3 (2023-08-09)
```
Comment (sokolov-group) [inline CHANGELOG]: Old date?

File: interface.py [hunk 2]
```diff
@@ -78,24 +99,26 @@ def __init__(self, mf, mc = None, opt_einsum = False):
...
         self.h1e_ao = mf.get_hcore()
+        self.v2e_ao = mf._eri
 
-        # TODO: replace exact 2e integrals with the DF integrals
-        self.v2e_ao = None
-        if mf._eri is None:
-            raise Exception("Out-of-core algorithm is not implemented for Pyscf")
-        else:
-            self.v2e_ao = mf._eri.copy()
+        self.with_df = None
```
Comment (sokolov-group) [inline interface.py:117]: Is this flag automatically turned on when RI basis is specified? Do we need it?

File: mr_adc_integrals.py [hunk 3]
```diff
@@ -57,6 +57,17 @@ def transform_2e_chem_incore(interface, mo_1, mo_2, mo_3, mo_4):
 
     return np.ascontiguousarray(v2e)
 
+def transform_2e_chem_incore_compacted(interface, mo_1, mo_2, mo_3, mo_4):
```
Comment (sokolov-group) [inline mr_adc_integrals.py]: Can't this be achieved using a flag in transform_2e_chem_incore? Is a separate function really necessary? Are there other examples of such practices in the code where flags can be used to remove redundancies?

File: CHANGELOG (hunk 1 above)
Comment (carlosevmoura) [reply CHANGELOG]: Date updated on a new commit

**Review by sokolov-group — APPROVED** [review]

**Pull request merged**